### PR TITLE
docs: move telemetry docs to Administration Guide

### PR DESCRIPTION
**Pull request by dkwon17**
Signed-off-by: David Kwon <dakwon@redhat.com>

## What does this pull request change
This PR removes che-server telemetry related content and moves the Telemetry section from `Extensions` to `Administration Guide`. ~~This PR does not change content, it only moves content.~~ This PR changes some titles and one sentence, see  https://github.com/eclipse-che/che-docs/pull/2246#issuecomment-1079475308.

**Important note:**
The Telemetry overview page and the Woopra telemetry plug-in page is still contains some details specific to che-server telemetry. I plan to update this content soon, either in this PR or in a different PR.

image

## What issues does this pull request fix or reference
https://github.com/eclipse/che/issues/21264

## Specify the version of the product this pull request applies to
Any version that supports devworkspace telemetry, which I believe should be version 7.27.x and above.

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [x] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]
LGTM, please address the PR check failure related to the unused image before merging

**Comment by l0rd** [comment]
What about moving it under: `administrator guide > configuring observability`?

Something that is not clear reading the doc is that this is not Che generic telemetry but:
- this is about che-theia based workspaces telemetry (no server side telemetry, not any workspace telemetry neither)
- although there is no documentation about it we also have chectl telemetry

**Comment by l0rd** [comment]
There is also [this pending PR](https://github.com/eclipse-che/che-docs/pull/2245) that puts the content of modules inside pages rather than using an include to partial for admin and user guides so you may want to adapt this PR to that (for example [architecture-overview using partials](https://github.com/eclipse-che/che-docs/blob/master/modules/administration-guide/pages/architecture-overview.adoc) vs [architecture-overview in one page](https://github.com/themr0c/che-docs/blob/modules-in-pages/modules/administration-guide/pages/architecture-overview.adoc)).

**Comment by ibuziuk** [comment]
> What about moving it under: administrator guide > configuring observability?

Sounds reasonable to have the ` administrator guide > configuring observability > telemetry` structure

> this is about che-theia based workspaces telemetry (no server side telemetry, not any workspace telemetry neither)

yeah, this is a good point. If we are not explicit enough that this is che-theia specific telemetry we need to add a warning about that. somewhere here in the  `How it Works`

image

**Comment by themr0c** [comment]
Please rebase on master. On master we can now have real content in pages (as planned on the main branch).

**Comment by l0rd** [comment]
> yeah, this is a good point. If we are not explicit enough that this is che-theia specific telemetry we need to add a warning about that. somewhere here in the How it Works

Yes and it should go beyond a warning. For example the outline:
- `Creating a Telemetry Plugin` should  be `Che-Theia Workspaces > Creating a Telemetry Plugin`
- `The Woopra Telemetry Plugin` should  be `Che-Theia  Workspaces > The Woopra Telemetry Plugin`
- `The Woopra Telemetry Plugin` should be displayed as the first item and `Creating a Telemetry Plugin` as the second (an admin will want to learn about the existing plugin first and then, if it doesn't match his use case, learn about how to create a custom one).

And in the Telemetry Overview:
```diff
- Telemetry is the explicit and ethical collection of operation data. By default, telemetry is not available in Eclipse Che, but there is an abstract API that allows enabling telemetry using the plug-in mechanism. This approach is used in the [Eclipse Che hosted by Red Hat](https://www.eclipse.org/che/docs/stable/hosted-che/hosted-che/) service where telemetry is enabled for every workspace.
+ Telemetry is the explicit and ethical collection of operation data. By default, telemetry is not available in Eclipse Che, but in the Che-Theia editor there is an abstract API that allows enabling telemetry using the plug-in mechanism and in the `chectl` command line tool usage data can be collected using segment. This approach is used in the [Eclipse Che hosted by Red Hat](https://www.eclipse.org/che/docs/stable/hosted-che/hosted-che/) service where telemetry is enabled for every Che-Theia workspace.
```

And the `Use Cases` and `How it Works` paragraphs should be moved from the `Overview` page a the `Che-Theia Workspaces` dedicated page where I have suggested to move the content above.

**Comment by dkwon17** [comment]
I made some new changes to this PR. This PR:
* deletes unnecessary image and examples
* deletes partials and adds the content directly to the pages
* moves the telemetry docs to be under the `Configuring observability` section
* telemetry overview and plugin docs are now under the new `Che-Theia workspaces` section
* adds @l0rd 's suggestion from this comment https://github.com/eclipse-che/che-docs/pull/2246#issuecomment-1078879531
* changed the `The Woopra Telemetry Plugin` title to `The Woopra telemetry plug-in` and the `Creating a Telemetry Plugin` to `Creating a telemetry plug-in` 

image

image

File: modules/administration-guide/pages/che-theia-workspaces.adoc [hunk 1]
```diff
@@ -1,11 +1,16 @@
-:parent-context-of-telemetry-overview: {context}
+[id="che-theia-workspaces"]
+// = Che-Theia workspaces
+:navtitle: Che-Theia workspaces
+:keywords: che-theia, workspaces
+:page-aliases: .:che-theia-workspaces
```
Comment (themr0c) [inline modules/administration-guide/pages/che-theia-workspaces.adoc]:
```suggestion
:page-aliases: extensions:che-theia-workspaces
```

File: modules/administration-guide/pages/creating-a-telemetry-plugin.adoc [hunk 2]
```diff
@@ -1,5 +1,11 @@
-[id="creating-a-telemetry-plugin-for-{devworkspace}s"]
-= Creating a telemetry plug-in for {devworkspace}s
+[id="creating-a-telemetry-plugin"]
+// = Creating a telemetry plugin
+:navtitle: Creating a telemetry plug-in
+:keywords: extensions, telemetry
+:page-aliases: .:creating-a-telemetry-plug-in
```
Comment (themr0c) [inline modules/administration-guide/pages/creating-a-telemetry-plugin.adoc]:
```suggestion
:page-aliases: extensions:creating-a-telemetry-plug-in
```

File: modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc [hunk 3]
```diff
@@ -1,6 +1,12 @@
 [id="the-woopra-telemetry-plugin"]
+// = The Woopra telemetry plugin
+:navtitle: The Woopra telemetry plug-in
+:keywords: extensions, telemetry
+:page-aliases: .:the-woopra-telemetry-plug-in
```
Comment (themr0c) [inline modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc]:
```suggestion
:page-aliases: extensions:the-woopra-telemetry-plug-in
```

**Review by themr0c — APPROVED** [review]

**Pull request merged**